sql/moz-fx-data-shared-prod/pocket/spoc_tile_ids/metadata.yaml

```diff
@@ -3,3 +3,7 @@ description: |-
   Lookup table of the currently registered tile IDs for Pocket sponsored content
 owners:
   - jklukas@mozilla.com
+workgroup_access:
+- role: roles/bigquery.dataViewer
+  members:
+  - workgroup:pocket/prefect
```

Comment:
```suggestion
  - workgroup:pocket/prefect
```

Reply:
I think the yaml lint is failing because of missing newlines at EOF.

Reply:
added the missing newlines